### PR TITLE
Lat lon

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

Filter sites on lat long
#111 

## How Has This Been Tested?

Filter sites on lat and long

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: pv_site_api/_db_helpers.py [hunk 1]
```diff
@@ -229,3 +231,36 @@ def check_user_has_access_to_sites(session: Session, auth: dict, site_uuids: lis
...
+
+def get_sites_from_user(session, user, latitude_longitude_max=None, latitude_longitude_min=None):
+    """
+    Get the sites for a user
+
+    Option to filter on latitude longitude max and min
+    """
+
+    # make LatitudeLongitudeLimits object
+    lat_lon_limits = LatitudeLongitudeLimits()
```
Comment (peterdudfield) [inline pv_site_api/_db_helpers.py:244]: seperater into seperat function

File: pv_site_api/_db_helpers.py [hunk 2]
```diff
@@ -229,3 +231,36 @@ def check_user_has_access_to_sites(session: Session, auth: dict, site_uuids: lis
                     f"does not have access to this site {site_uuid}. "
                     f"User has access to {site_uuids}",
                 )
+
+
+def get_sites_from_user(session, user, latitude_longitude_max=None, latitude_longitude_min=None):
```
Comment (rachel-labri-tipton) [inline pv_site_api/_db_helpers.py]: hi @peterdudfield, just a general question here, why don't functions like this one live in the `pv site datamodel`? for instance, if it did live in the data model (or some version of it, i realise the `LatLonLimits` class is particular to the api), then this function could be available for the OCF Dashboard as well.

Comment (peterdudfield) [reply pv_site_api/_db_helpers.py]: short answer is: It should be in `pv site datamodel`, Ill move it over

**Review by rachel-labri-tipton — APPROVED** [review]

File: pv_site_api/_db_helpers.py (hunk 2 above)
Comment (peterdudfield) [reply pv_site_api/_db_helpers.py]: I've put it here - https://github.com/openclimatefix/pv-site-datamodel/issues/83

**Pull request merged**